main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -260,6 +260,7 @@ class Flix {
     "Abort.flix" -> LocalResource.get("/src/library/Abort.flix"),
     "Clock.flix" -> LocalResource.get("/src/library/Clock.flix"),
     "Http.flix" -> LocalResource.get("/src/library/Http.flix"),
+    "HttpWithResult.flix" -> LocalResource.get("/src/library/HttpWithResult.flix"),
```

Comment:
Can we do this in separate PR?

This one is supposed to fix a bug.

Reply:
Or, well, there's sort of a circular dependency, since this PR won't compile without this, and introducing `HttpWithResult` is weird when there's no error handling in it.

Reply:
OK. Rename the PR then